### PR TITLE
'images' renamed as 'artifacts' - ready for review; do not merge (should be merged at the same time with other parts of MEN-847 task)

### PR DESCRIPTION
**Pull request by kjaskiewiczz**
Changes:
API docs updated.
Endpoint changed.
Deployment status 'noimage' renamed as 'noartifact'.

Part of MEN-847 issue
@bboozzoo @maciejmrowiec @mchalski

### CODE REVIEW

**Comment by maciejmrowiec** [comment]
@michaelatmender  probably UI needs to follow up on noimage -> noartifact change

**Comment by kjaskiewiczz** [comment]
@maciejmrowiec I've updated PR to UI
I'll wait for @michaelatmender response

**Comment by maciejmrowiec** [comment]
Awesome, thanks for taking care of this!

File: docs/integrations_api.yml [hunk 1]
```diff
@@ -319,32 +319,32 @@ paths:
           type: string
         - name: firmware
```
Comment (maciejmrowiec) [inline docs/integrations_api.yml]: should we call this one `artifact` as well ?

Comment (kjaskiewiczz) [inline docs/integrations_api.yml]: We could.
There is also paragraph:
"List known firmware artifacts"
Should we change this to:
"List known artifacts"?

Comment (kjaskiewiczz) [inline docs/integrations_api.yml]: @maciejmrowiec ^^

Comment (maciejmrowiec) [inline docs/integrations_api.yml]: This so, we renamed firmware to artifacts in other places.

Comment (kjaskiewiczz) [inline docs/integrations_api.yml]: updated

Comment (kjaskiewiczz) [inline docs/integrations_api.yml]: docs + impelemntation

**Pull request merged**